src/tpm2tss-genkey.c

```diff
@@ -50,7 +50,8 @@ char *help =
     "Arguments:\n"
     "    <filename>      storage for the encrypted private key\n"
     "Options:\n"
-    "    -a, --alg       public key algorithm (rsa, ecdsa) (default: rsa)\n"
+    "    -a, --alg       public key algorithm (rsa, ecdsa, aes) (default: rsa)\n"
```

Comment:
If we only support persistent keys for now, then all changes to this file should be removed.